### DISCUSSION TITLE
Getting added as a "Desktop" option

### DISCUSSION THREAD

**Discussion by ab599597** [WalletConnect/walletconnect-monorepo#3546] [Developer Support]
Hey all, I know this maybe the wrong location to ask this but after reviewing the Discord, this was my best assumption.

What is the process for new desktop / browser extension wallets to get added as an option in Wallet Connect.

Below is the location in question, thanks
<a href="https://ibb.co/hmkRFv9"><img src="https://i.ibb.co/kSpg2FB/Screenshot-2023-08-25-at-11-02-57-AM.png" alt="Screenshot-2023-08-25-at-11-02-57-AM" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>upload pic for free</a><br />

**Comment by ab599597** [comment]
Following up on this

**Comment by boidushya** [comment]
In your cloud dashboard, head over to the explorer section and add a deeplink for your desktop wallet. You will also need to implement this feature on your wallet's side to take effect.
<img width="778" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/26746725/e1329c37-a264-4ab9-a707-fa0e3eaa960e">
